contrib/slurm-benchmark.py

```diff
@@ -56,6 +56,13 @@ def create_argument_parser(self):
             default="./",
             help="The directory where temporary directories can be created for use within singularity.",
         )
+        slurm_args.add_argument(
+            "--retry-killed",
+            dest="retry",
+            type=int,
+            default="0",
+            help="Retry killed jobs this many times. Use a negative number for unbounded retry attempts.",
```

Comment:
I would recommend just `-1` for unbounded. This is similar to what BenchExec does for other parameters.

Reply:
Sure, fixed.